D1673/P1673.md

```diff
@@ -7318,11 +7315,12 @@ sum_of_squares_result<T> vector_sum_of_squares(
 <i>[Note:</i> These functions correspond to the LAPACK function `xLASSQ`.
 <i>-- end note]</i>
 
-[1]{.pnum} *Preconditions:*
+[1]{.pnum} *Mandates:*
 
-  * [1.1]{.pnum} `T` shall be *Cpp17MoveConstructible* and *Cpp17LessThanComparable*, and
+  * [1.1]{.pnum} `T` meets both the *Cpp17MoveConstructible* and
+    *Cpp17LessThanComparable* requirements, and
 
-  * [1.2]{.pnum} `abs(x[0])` shall be convertible to `T`.
+  * [1.2]{.pnum} `abs(x[0])` is convertible to `T`.
 
 [2]{.pnum} *Constraints:* For all `i` in the domain of `v`,
```

Comment:
This Constraint is left over from the "mathematical expression of the algorithm" R10 wording that R11 replaced.  I'll reword it.

Reply:
I've reworded this in PR https://github.com/ORNL/cpp-proposals-pub/pull/354 .